src/Type/Object/PostObject.php

```diff
@@ -209,12 +209,6 @@ public static function register_post_object_types( $post_type_object ) {
 	public static function get_post_object_fields( $post_type_object ) {
 		$single_name = $post_type_object->graphql_single_name;
 		$fields      = [
-			'id'                => [
```

Comment:
So, you're deleting this because it is _exact_ same as the Interface definition and will inherit the interface definition of the field, but you're leaving the `id` field on `TermObject.php` because the description is a bit different?

Reply:
Yea. pretty much :man_shrugging:. We can change them all, or do a deep merge to simple overwrite the `description` instead. What do you think?